_posts/2016-05-20-drupalcon-nola-recap.md

```diff
@@ -0,0 +1,118 @@
+
+After sessions closed for the day, we had a nice dinner with friends from [DevCollaborative](http://devcollaborative.com/about-us), [Advomatic](https://www.advomatic.com/our-team/sarah-german)
+and made our way on over to the [Lullabot party](https://events.drupal.org/neworleans2016/lullabots-8th-annual-drupalcon-party). The theme of meeting those who we only digitally knew, while making new friends certainly continued at the Lullabot party. It was a great time!
+
+### Thursday - the last day of sessions
+
+Thursday was the final day of sessions, and of course the [Drupal 6 burial party](https://events.drupal.org/neworleans2016/kalamuna-four-kitchens-present-drupal-6-mourning-party) - #SoCheeky. The sessions I attended were all quite pertinent to me. In my talk, I reference the lack of an official release for an e-commerce platform for Drupal 8 is one reason for slowed adoption compared to Drupal 7. The Commerce guys did elicit a lot of applause in [their session](https://events.drupal.org/neworleans2016/sessions/creating-stores-drupal-commerce-drupal-8) about what _is coming soon_ for Drupal Commerce's official Drupal 8 release, but were only willing to commit to an ready by the "end of summer" which left [some people](https://youtu.be/EQxA3VYal54?t=3161) chomping at the bit for an official release that is still at arm's length.
+
+ Zivtech's Co-founder/CTO [Jody Hamilton](https://www.zivtech.com/about/team/jody-hamilton)'s ["Grow your own"](https://events.drupal.org/neworleans2016/sessions/grow-your-own) presentation about how to create a culture of recruitment and training within was another near-and-dear topic as we had just hired our second intern ([the wonderful Ro](http://savaslabs.com/team/ro-wang/))! The presentation had some good lessons learned and valuable takeaways if not only entertaining to the frequency of the reference to Marijuana (see 3 seconds into the presentation). Thanks for the good info Jody!
```

Comment:
Just a couple thoughts I had on `The presentation had some good lessons learned and valuable takeaways if not only entertaining to the frequency of the reference to Marijuana (see 3 seconds into the presentation).` (1) Is marijuana capitalized, (2) is it worth evening mentioning this? (3) the "if not only entertaining" read a little funny to me. I'm fine with it as is though.